style.css

```diff
@@ -0,0 +1,123 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  width: 375px;
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 50px;
+  background: #3c3a39;
+}
+
+.logo {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  display: flex;
+  position: relative;
+  width: 77px;
+  height: 24px;
+  left: 12px;
+  top: 0%;
+  color: white;
+}
+
+.menu-btn {
+  position: relative;
+  top: 10%;
+  right: 30px;
+}
+
+.menu {
+  width: 35px;
+  height: 5px;
+  background-color: white;
+  margin: 6px 0;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
+  width: 375px;
+  height: 590px;
+  background: #1c1a19;
+  top: 50px;
+  left: 0;
+  background-image: url("background.png");
+  background-repeat: no-repeat;
+  background-position: right 0 bottom -50px;
+  background-size: 35%;
+}
```

Comment:
- [ ] It seems that you use the wrong background for the headline or that background doesn't exist in your repo. 
- ### your design: 
![image](https://user-images.githubusercontent.com/71874425/158898991-77ed7812-d2db-4475-aab5-cf33067d7c33.png)
Make sure that your design matches one of the designs provided in these [Figma designs](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) from Microverse.
- [ ] Please make sure the section has the margins (top, bottom, left, right) defined in the design

Reply:
Thank you very much for the review, I have tried to fix the errors. I hope I'm correct.
and my design looks same with the figma design when I open it on live server. I've sent you screen shoot.

![image](https://user-images.githubusercontent.com/93576145/158906837-8e9c6e0b-3f76-435c-8dc5-c16b9de03447.png)